Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM phusion/baseimage:0.9.19
+
+# Use baseimage-docker's init system.
+CMD ["/sbin/my_init"]
+
+# Directory containing ADAGE server source code
+ENV ADAGE_SRC=adage
+# Directory used in container
+ENV ADAGE_SRV=/srv
+# Directory where ADAGE code lives in the container
+ENV ADAGE_SRVSRC=$ADAGE_SRV/$ADAGE_SRC
+
+RUN apt-get update && apt-get install -y \
+  python \
+  python-pip \
+  python-psycopg2
+
+# Create required directories
+WORKDIR $ADAGE_SRV
+RUN mkdir media static logs
```

Comment:
I have no idea. If nothing I can remove this. @mhuyck : are we using this feature?